src/include/daos/common.h

```diff
@@ -202,6 +202,15 @@ daos_getntime_coarse(void)
 	return (tv.tv_sec * NSEC_PER_SEC + tv.tv_nsec); /* nano seconds */
 }
 
+static inline uint64_t
+daos_getutime(void)
```

Comment:
minor: but maybe add this to gurt?

Reply:
Yeah, I thought about the same thing. There are already daos_get_ntime, daos_getntime_coarse, daos_gettime_coarse in this file though (look immediately above and below).